metrics/api/v1/model_handlers.go

```diff
@@ -60,6 +60,10 @@ type clusterMetricsFetcher interface {
 	availablePodMetrics(request *restful.Request, response *restful.Response)
 	podMetrics(request *restful.Request, response *restful.Response)
 
+	namespacePVCList(request *restful.Request, response *restful.Response)
```

Comment:
I don't think this is necessary.  The model API is more-or-less deprecated (see #1898), and we don't really want to encourage its use by adding new stuff to it.